multipath/custom.conf

```diff
@@ -0,0 +1,6 @@
+# Custom configuration for multipathd
```

Comment:
My suggestion with this file is to add it by default on systems and tell users about it in the main multipath.conf (as well as documentation), so that they have an easy entry point for custom configuration. The file would be marked `%config(noreplace)` in the spec file so that updates don't overwrite user changes in it.